internal/rbd/snapshot.go

```diff
@@ -118,6 +121,16 @@ func (rbdSnap *rbdSnapshot) toVolume() *rbdVolume {
 	}
 }
 
+func (rbdSnap *rbdSnapshot) ToCSI(ctx context.Context) (*csi.Snapshot, error) {
```

Comment:
Let's make use of it [here](https://github.com/ceph/ceph-csi/blob/devel/internal/rbd/controllerserver.go#L1238) ?

Reply:
oh, sure, this function actually was not required to be included, it is part of my VolumeGroupSnapshot work. I'll include it here so that it all gets a little cleaner.